PocketKit/Tests/SyncTests/PocketSourceTests.swift

```diff
@@ -625,3 +625,62 @@ extension PocketSourceTests {
         }
     }
 }
+
+// MARK: - Search Term
+extension PocketSourceTests {
+    func test_searchItems_searchFree() throws {
+        let source = subject()
+
+        var savedItems = (1...2).map {
+            space.buildSavedItem(
+                remoteID: "saved-item-\($0)",
+                item: space.buildItem(title: "saved-item-\($0)")
+            )
+        }
+        savedItems.append(
+            space.buildSavedItem(
+                remoteID: "test-item",
+                item: space.buildItem(title: "item")
+            )
+        )
+        _ = createItemsWithTags(2)
+
+        try space.save()
+
+        let results = source.searchSaves(search: "saved")
+        XCTAssertTrue(results?.count == 2)
+
+        let tagResults = source.searchSaves(search: "tag")
+        XCTAssertTrue((tagResults?.isEmpty) != nil)
```

Comment:
if we are testing a free user here, shouldn't `tagResults` be empty?

Reply:
yup.